### DISCUSSION TITLE
RAC Button has no aria-disabled available

### DISCUSSION THREAD

**Discussion by brenotx** [adobe/react-spectrum#6571] [Q&A] [answered]
Hi, I'm trying to implement an `isPending` property to the RAC Button component. The problem is that I can't pass down the `aria-disabled` property.  

The pending state is a very common pattern. React Spectrum implements using the `useButton` hook. I wonder why the RAC Button doesn't expose `aria-disabled` and if there is a solution without using react-aria hooks.

**Comment by dannify** [comment, marked as answer]
This feature hasn't been added to RAC Button yet. It is somewhat new in React Spectrum and just hasn't made it everywhere else. I am hoping to look at adding it in the next few months depending on some other priorities. We are also open to contributions!
